timesketch/app.py

```diff
@@ -91,6 +91,16 @@ def create_app(config=None):
                          '$ openssl rand -base64 32\n\n')
         sys.exit()
 
+    # Support old style config using Elasticsearch as backend.
+    # TODO: Deprecate the old ELASTIC_* config in 2023.
+    if not app.config.get('OPENSEARCH_HOST'):
+        sys.stderr.write('Deprecated config field found: ELASTIC_HOST\n')
+        app.config['OPENSEARCH_HOST'] = app.config.get('ELASTIC_HOST')
+
+    if not app.config.get('OPENSEARCH_PORT'):
+        sys.stderr.write('Deprecated config field found: ELASTIC_PORT\n')
```

Comment:
Here as well, I would warn the user which field is missing at least and not just telling what was found.

Reply:
Done